modules/Hoot/tools/folderTree.js

```diff
@@ -643,7 +654,7 @@ export default class FolderTree extends EventEmitter {
             selected = data.selected || false,
             isOpen   = data.state === 'open';
 
-        if ( data.type === 'dataset' ) {
+        if ( data.type === 'dataset' || data.type === 'folder' && d3.event.ctrlKey ) {
```

Comment:
are there just two `data.type` values?  folder and dataset?
is there still a need for this || conditional?

Reply:
I did this to prevent duplication of code which listens for a ctrl key event on a folder